docs/docs/APIdocs/layers/nn/Containers/Container.md

```diff
@@ -0,0 +1,7 @@
+## Container ##
+
+Container is an abstract AbstractModule class which
```

Comment:
"an abstract AbstractModule class" doesn't seem clear to users as there is no link to "AbstractModule", maybe "a subclass of abstract class AbstractModule" ?